lib/seed/facility_seeder.rb

```diff
@@ -74,9 +71,8 @@ def weighted_facility_size_sample
       FACILITY_SIZE_WEIGHTS.max_by { |_, weight| rand**(1.0 / weight) }.first
     end
 
-    def facilities_per_block
-      max_number_facilities_per_block = (number_of_facilities_per_facility_group / MAX_FACILITIES_PER_BLOCK).ceil
-      config.rand_or_max(1..max_number_facilities_per_block)
+    def max_number_of_blocks_per_facility_group
+      2
```

Comment:
I've put this into a method since it's likely a parameter we'll want to get fancy with soon. Let me know if you'd like me to just shove it back into the method below instead.